File: API/Features/Reservations/Ledgers/ViewModels/Report/LedgerPortGroupVM.cs
namespace API.Features.Reservations.Ledgers {

    public class LedgerPortGroupVM {

        public int Adults { get; set; }
        public int Kids { get; set; }
        public int Free { get; set; }
        public int TotalPax { get; set; }
        public int TotalPassengers { get; set; }
        public int TotalNoShow { get; set; }
        public bool HasTransfer { get; set; }

    }

}

File: FrontEnd/src/app/features/reservations/ledgers/classes/view-models/list/ledger-port-group-vm.ts
import { LedgerPortVM } from './ledger-port-vm'

export interface LedgerPortGroupVM {

    port: LedgerPortVM
    hasTransferGroup: HasTransferGroupVM[]
    adults: number
    kids: number
    free: number
    totalPax: number
    totalPassengers: number
    totalNoShow: number

}

export interface HasTransferGroupVM {

    hasTransfer: boolean
    adults: number
    kids: number
    free: number
    totalPax: number
    totalPassengers: number
    totalNoShow: number

}